src/main/java/com/ita/if103java/ims/dto/UserDto.java

```diff
@@ -0,0 +1,215 @@
+package com.ita.if103java.ims.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.ita.if103java.ims.dto.transfer.ExistData;
+import com.ita.if103java.ims.dto.transfer.NewData;
+import com.ita.if103java.ims.dto.transfer.NewDataAdmin;
+import com.ita.if103java.ims.dto.transfer.NewDataWorker;
+import com.ita.if103java.ims.entity.Role;
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Component
+public class UserDto implements Serializable {
+
+    @Null(groups = {NewDataAdmin.class},
+        message = "This field must be filled with the auto-generator during registration")
```

Comment:
For now just think about localization and create task in the trello, to add localization with low priority

Reply:
I' ve already created this task in the trello.

Reply:
To tell the truth, I am not sure whether my understanding of meaning "localization with low priority" is right.

Reply:
low priority means to do when all other important tasks are done